src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -355,13 +355,22 @@ class LicenceService(
           licenceId = licenceId,
           username = request.username,
           fullName = request.fullName,
+          eventType = getAuditEventType(request),
           summary = summaryText,
           detail = detailText,
         )
       )
     )
   }
 
+  private fun getAuditEventType(request: StatusUpdateRequest): AuditEventType {
+    return if (request.username == "SYSTEM") {
+      AuditEventType.SYSTEM_EVENT
+    } else {
+      AuditEventType.USER_EVENT
+    }
+  }
+
```

Comment:
This username _**can**_ be `SYSTEM` because thats what we pass into the endpoint